### PR TITLE
IT-793: Update Sceptre Hooks

### PR DESCRIPTION
**Pull request by cascadianblue**
Switch all of the existing config files to use the new before_launch Sceptre hook, instead of the two before_create and before_update Sceptre hooks that they used before.

### CODE REVIEW

File: config/prod/xschildw-shinyproxy.yaml [hunk 1]
```diff
@@ -40,9 +40,7 @@ parameters:
 
 # For CI system (do not change)
 hooks:
-  before_create:
-    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-linux-v1.j2 --create-dirs -o templates/remote/managed-ec2-linux-v1.j2"
-  before_update:
+  before_launch:
     - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-linux-v1.j2 --create-dirs -o templates/remote/managed-ec2-linux-v1.j2"
```
Comment (cascadianblue) [inline config/prod/xschildw-shinyproxy.yaml:44]: @zaro0508 I believe this is the file that you said I had missed in your review, but it looks updated to me. Is there something I'm missing?

**Comment by cascadianblue** [comment]
I'm looking back at my branch, and I don't think I did miss that one, actually: 
https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/352/commits/939af4a65d75628e08e37c2211aa656a89a6eb03#diff-7197abc660d476184371d86c4a974156

**Comment by zaro0508** [comment]
sorry i provided the wrong link, it's this one.. https://github.com/Sage-Bionetworks/sandbox-provisioner/blob/master/config/prod/xschildw-packertest.yaml

**Comment by cascadianblue** [comment]
That appears to be the same link

**Comment by zaro0508** [comment]
please check again @cascadianblue, it's not.

**Pull request merged**